ergoCubSN001/estimators/wholebodydynamics.xml

```diff
@@ -72,8 +72,8 @@
       <!-- ft -->
       <elem name="l_leg_ft_sensors">left_leg-FT_remapper</elem>
       <elem name="r_leg_ft_sensors">right_leg-FT_remapper</elem>
-      <elem name="l_arm_ft_sensors">left_arm-eb2-j0_1-strain</elem>
-      <elem name="r_arm_ft_sensors">right_arm-eb1-j0_1-strain</elem>
+<!--      <elem name="l_arm_ft_sensors">left_arm-eb2-j0_1-strain</elem>
+      <elem name="r_arm_ft_sensors">right_arm-eb1-j0_1-strain</elem> -->
```

Comment:
Can we proceed anyway with merging?
The fork overrides upstream so I reckon that upstream will receive a new update when the FT will be fixed.

Reply:
Ok! Then, I'll wait 'till the fork branch gets updated in this respect.